components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -355,7 +326,7 @@ ${helpers.predefined_type(
     }
 
     pub fn parse<'i, 't>(_context: &ParserContext, input: &mut Parser<'i, 't>)
```

Comment:
nit: Indentation here still looks somewhat weird, feel free to switch to block indentation entirely if you want.